index.bs

```diff
@@ -7,3 +7,514 @@ URL: https://netwerk-digitaal-erfgoed.github.io/requirements-network-of-terms/
+In order to quickly gain insight into problems, the Network of Terms keeps track of the status of terminology sources.
+The status makes clear whether or not the sources are reachable.
+If this is not the case, the application manager of the Network of Terms can take action,
+for example by contacting the owner of the source or by informing his users.
+
+The Network of Terms provides a status page where the status of each terminology source is displayed. The status page is publicly accessible.
+
+The application manager of the Network of Terms can add, edit and remove terminology sources {#18}
+--------------------
+**Priority**: Should have
+
+Terminology sources have their own protocols and data structures for querying and expressing terms.
+The Network of Terms knows about these, as well as how the information about terms is mapped
+into the uniform, source-independent data structure of the Network of Terms.
+
+The application manager of the Network of Terms maintains the configuration about sources and mappings.
+He can add, edit and remove sources without the intervention of a software developer.
+
+<div class="note">
+    **Discussion**: this requirement needs further eloboration.
+    For example: is it sufficient if an application manager can modify the configuration files of the Network of Terms?
+    Or does he need a user interface for this?
+</div>
+
+The Network of Terms has developer-friendly documentation about its interfaces {#19}
+--------------------
+**Priority**: Must have
+
+Software developers of systems of users want to be able to implement the Network of Terms easily.
+Therefore the Network of Terms must have documentation that is accessible to developers.
+The documentation describes the interfaces of the Network of Terms, such as the query language, data format and metadata schemas.
+The documentation must be generated from the source code of the Network of Terms so that it is always up-to-date, e.g. with OpenAPI.
+
+The Network of Terms publishes machine-readable information about its interfaces {#20}
+--------------------
+**Priority**: Should have
+
+The Network of Terms is primarily used by systems of users, not directly by users.
+Therefore the Network of Terms should not only have documentation for software developers (see [requirement 19](#19)),
+but also publish machine-readable information about how its interfaces can be used by systems.
+This information makes the Network of Terms self-descriptive.
+The information makes clear, for example, which interfaces there are and which HTTP methods they support (such as POST or GET).
+
+The Network of Terms has an availability of 99% {#21}
+--------------------
+**Priority**: Must have
+
+The Network of Terms is a generic facility that can be used at all times by all participants in the Dutch Digital Heritage Network.
+Therefore the Network of Terms must have a high availability of 99%.
```

Comment:
That’s not _really_ high (99% availability = 3.65 days downtime per year) but fine with me.

Reply:
Yes, good point. I'll add a discussion note about this.